modules/odometry_and_time.py

```diff
@@ -12,12 +12,29 @@ class OdometryAndTime:
     """
     Contains odometry/telemetry and timestamp.
     """
+    __create_key = object()
+
+    @classmethod
+    def create(cls, 
+               position: drone_odometry.DronePosition, 
+               orientation: drone_odometry.DroneOrientation) -> "tuple[bool, MergedOdometryDetections | None]":
+        """
+        Constructor sets timestamp to current time.
+        position: Latitude, longitude in decimal degrees and altitude in metres.
+        orientation: Yaw, pitch, roll following NED system (x forward, y right, z down).
```

Comment:
Not needed, `DronePosition` and `DroneOrientation` already describe this. This class exists only to hold them together with a timestamp.